metricbeat/module/system/process/helper.go

```diff
@@ -48,6 +49,11 @@ func newProcess(pid int) (*Process, error) {
 		return nil, fmt.Errorf("error getting process state for pid=%d: %v", pid, err)
 	}
 
+	exe := sigar.ProcExe{}
+	if err := exe.Get(pid); err != nil {
```

Comment:
This will require more error handling. Some platforms do not implement `sigar.ProcExe` and they should return `ErrNotImplemented`. We don't want to break all process metric collection if this error occurs. It would be good to double check the implementations in gosigar to see if there are other errors that can occur in normal operation (like a permission error if metricbeat is run by a non-root user).